Gems/Atom/RHI/Vulkan/Code/Source/RHI/Device.cpp

```diff
@@ -248,6 +248,7 @@ namespace AZ
 
             // If we are running Vulkan >= 1.2, then we must use VkPhysicalDeviceVulkan12Features instead
             // of VkPhysicalDeviceShaderFloat16Int8FeaturesKHR or VkPhysicalDeviceSeparateDepthStencilLayoutsFeaturesKHR.
+            auto timelineSemaphore = physicalDevice.GetPhysicalDeviceTimelineSemaphoreFeatures();
```

Comment:
This line separates the comments above from the `if` that they document. Please move the line further up.